ITEMS.md

```diff
@@ -104,3 +104,4 @@ The following is a list of currently supported IotMs. Note that because this is
+| [Tiny Stillsuit](https://kol.coldfront.net/thekolwiki/index.php/Tiny_stillsuit)                                                 | Equip to quantized familiar to boost item%                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     | Parse actual item% from stillsuit to determine if other familiars would offer higher item drops.                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                |
```

Comment:
The items listed here are sorted by (1) IotMs, (2) IotYs and (3) others, with each subgroup organized by chronological order for ease of reference. This entry should be parked between parka and designer sweatpants.

We should replace quantized familiar with [quantum entangler](https://kol.coldfront.net/thekolwiki/index.php/Quantum_entangler) (with the hyperlink), and I would remove the "Parse actual item% from stillsuit to determine if other familiars would offer higher item drops" for now - it's more of an optimization problem than simply not supporting a feature that the item provides